.travis.yml

```diff
@@ -5,14 +5,11 @@ sudo: false
 
 language: generic
 
-services:
-    - docker
```

Comment:
Don't we still need this for the builds?

Reply:
Nope, docker is part of the image. No need to call this.

Reply:
Oh! I guess that's new!